shared/src/components/exercise/index.cjsx

```diff
@@ -104,7 +104,7 @@ ExerciseMixin =
     {parts} = @props
     step = _.last(parts)
 
-    related_content = step.related_content if related
+    related_content = step.related_content # if related
 
```

Comment:
im pretty sure this is no longer being used...this was with the multi-part questions to prevent the "comes from" from being after each part.